src/cli/utils/loadCli.ts

```diff
@@ -84,7 +84,7 @@ export async function loadCli() {
       await downloadCli()
     }
   } catch (err) {
-    vscode.window.showErrorMessage('Error downloading DevCycle CLI')
+    vscode.window.showErrorMessage('Error downloading DevCycle CLI: ' + err)
```

Comment:
is there anyway to log the full error only if the debug logs are enabled?

Reply:
this error should be important enough to know the full context even without debug logs imo